clients/js-legacy/package.json

```diff
@@ -20,12 +20,14 @@
         "format:fix": "prettier --write src test",
         "test": "start-server-and-test 'solana-test-validator --bpf-program recr1L3PCGKLbckBqMNcJhuuyU1zgo8nBhfLVsJNwr5 ../../target/deploy/spl_record.so --reset --quiet' http://127.0.0.1:8899/health 'mocha test'"
     },
+    "peerDependencies": {
+        "@solana/web3.js": "^1.95.5"
+    },
     "devDependencies": {
         "@types/chai": "^5.0.1",
         "@types/node": "^22.10.1",
         "@types/mocha": "^10.0.10",
-        "@solana/web3.js": "^1.95.5",
-        "bigint-buffer": "^1.1.5",
+        "@solana/codecs-numbers": "^2.0.0",
```

Comment:
This will need to be in the normal dependencies, and not devDependencies

Reply:
Fixed in [d7ae3f1](https://github.com/solana-program/record/pull/20/commits/d7ae3f13f2eeff6cdcbf3723a468471916aa3cbe).